lib/vssh/libssh.c

```diff
@@ -1292,8 +1302,11 @@ static CURLcode myssh_statemach_act(struct Curl_easy *data, bool *block)
                  CURL_READFUNC_ABORT return code still aborts */
               failf(data, "Failed to read data");
               MOVE_TO_ERROR_STATE(CURLE_FTP_COULDNT_USE_REST);
+              break;
             }
           } while(passed < data->state.resume_from);
+          if(rc)
```

Comment:
nit: Maybe do `rc != SSH_OK` (see line 776)

Reply:
Right, but I prefer `if(rc)` than the other ways as `SSH_OK` is defined to zero.If you search in the libssh.c file you can also see a lot of `if(rc != 0)` checks, so it's not even consistently done already before.